presubmit-service/check-pr.ts

```diff
@@ -0,0 +1,12 @@
+import * as shell from 'shelljs';
+
+/** Determine if the PR can apply to a repo's targeted branch. */
+export const checkCherryPickibilityOfPr = (org: string, repo: string, pr: string, branch: string) => {
```

Comment:
How about `getCherryPickViability` or `getCherryPickConflictStatus`? I don't think you need "ofPr" in the function name since the PR is one of the arguments.